packages/types/src/providers.ts

```diff
@@ -0,0 +1,36 @@
+export type FacebookOauthProvider = 'facebook';
```

Comment:
🔧 @nikosdouvlis Let's split these in to files, one called `Oauth` and another called `web3` so that it's more accurate.

Reply:
Done, thanks